src/api/eth/header.rs

```diff
@@ -32,12 +39,37 @@ pub async fn handle(block: web::Path<String>, shared: web::Data<ShadowShared>) -
                 .unwrap_or_default(),
         )
     };
+    
+	let proof = ProofResp {
+		header: EthHeader::get(&shared.client, num)
+		.await
+		.unwrap_or_default()
+		.into(),
+		mmr_root: format!("0x{}", root),
+	};
+
+	let confirmations = get_confirmations(num).await.unwrap_or(0);
+
+	web::Json(Resp {
+		proof: proof,
+		confirmations: confirmations
+	})
```

Comment:
This resp will break the guard API of dj

Reply:
yes, maybe put this to dj is a better choice.